frontend/src/scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow.tsx

```diff
@@ -243,7 +243,7 @@ export function ActionFilterRow({
     const suffix = typeof customRowSuffix === 'function' ? customRowSuffix({ filter, index, onClose }) : customRowSuffix
 
     const propertyFiltersButton = (
-        <IconWithCount count={filter.properties?.length || 0} showZero={false}>
+        <IconWithCount key="property-filter" count={filter.properties?.length || 0} showZero={false}>
```

Comment:
what does the addition of the `key`s do in this file?

Reply:
It was to remove React errors clogging up the console while I was trying to debug some other parts. I think it has to do with the fact that the buttons are all rendered in a loop further down